src/Service/CloudWatch/src/Result/ListMetricsOutput.php

```diff
@@ -26,13 +26,47 @@ class ListMetricsOutput extends Result implements \IteratorAggregate
     private $nextToken;
 
     /**
-     * Iterates over Metrics.
+     * If you are using this operation in a monitoring account, this array contains the account IDs of the source accounts
+     * where the metrics in the returned data are from.
+     */
+    private $owningAccounts;
+
+    /**
+     * Iterates over Metrics and OwningAccounts.
      *
-     * @return \Traversable<Metric>
+     * @return \Traversable<Metric|string>
```

Comment:
Changing the return type signature is a BC break.

How should we deal with this?
- ignore the BC break like AWS do
- release this change in a major version

any opinions @Nyholm?

Reply:
Let's release this change in a new major version of CloudWatch.

Reply:
In that case how to we deal with migration path and providing deprecations in the generated code :( ?